src/Simplex/Chat/Library/Commands.hs

```diff
@@ -4256,10 +4349,22 @@ chatCommandP =
         CTLocal -> pure $ ChatName CTLocal ""
         ct -> ChatName ct <$> displayNameP
     chatNameP' = ChatName <$> (chatTypeP <|> pure CTDirect) <*> displayNameP
-    chatRefP = ChatRef <$> chatTypeP <*> A.decimal
+    chatRefP =
+      (A.char '@' $> ChatRef CRTDirect <*> A.decimal)
+        <|> groupChatRefP
+        <|> (A.char '*' $> ChatRef CRTLocal <*> A.decimal)
+        <|> (A.char ':' $> ChatRef CRTContactConnection <*> A.decimal)
+    groupChatRefP = do
+      gId <- A.char '#' *> A.decimal
+      gcs <- A.space *> gcScopeP <|> pure GCSGroup
+      pure $ ChatRef (CRTGroup gcs) gId
     sendRefP =
       (A.char '@' $> SRDirect <*> A.decimal)
-        <|> (A.char '#' $> SRGroup <*> A.decimal <*> optional (" @" *> A.decimal))
+        <|> (A.char '#' $> SRGroup <*> A.decimal <*> (A.space *> gcScopeP <|> pure GCSGroup))
+    gcScopeP =
+      ("@group" $> GCSGroup)
```

Comment:
This is unnecessarily complex syntax. Default scope can remain as is - `#123`

For support scope I suggest `#123(456)` and `#123()`.

The proposed syntax does not keep it together.

Reply:
I think we should use "support" for support here, to easier add new scope if required (e.g. admins to each other, or direct message to other member), and not have to change other apis or leave their syntax as "more general". also it's syntax for ui so why does it matter that it's a bit longer? also `#123()` is completely unintelligible